WearBuildConfig/mobile/build.gradle

```diff
@@ -0,0 +1,71 @@
+apply plugin: 'com.android.application'
+apply from: "$rootProject.projectDir/WearBuildConfig/shared-config/android-signing.gradle"
+
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.0"
+
+    defaultConfig {
+        applicationId "com.novoda.demo.wearbuildconfig"
+        minSdkVersion 16
+        targetSdkVersion 25
+        versionCode 1
+        versionName "1.0"
+
+        buildConfigField 'boolean', 'PLAY_SERVICES', "true"
+    }
+
+    buildTypes {
+        debug {
+            signingConfig signingConfigs.debug
+        }
+
+        release {
+            signingConfig signingConfigs.release
+        }
+
+        qaRelease.initWith(buildTypes.release)
+        qaRelease {
+            signingConfig signingConfigs.qa
+        }
+    }
+
+    productFlavors {
+
+        flavorDimensions 'environment', 'playServices'
+
+        noPlayServices {
+            dimension 'playServices'
+
+            buildConfigField 'boolean', 'PLAY_SERVICES', 'false'
+        }
+
+        playServices {
+            dimension 'playServices'
+            //uses default PLAY_SERVICES value
+        }
+
+        preProduction {
+            dimension 'environment'
+        }
+
+        production {
+            dimension 'environment'
+        }
+    }
+}
+
+configurations {
+    productionPlayServicesReleaseWearApp
+    preProductionPlayServicesReleaseWearApp
+    productionPlayServicesQaReleaseWearApp
+    preProductionPlayServicesQaReleaseWearApp
+}
+
+dependencies {
+    productionPlayServicesReleaseWearApp project(path: ':WearBuildConfigWear', configuration: 'productionRelease')
```

Comment:
I'm lost. Above you defined 4 additional configurations, but they've not be used anywhere else, right? Maybe this is material for the (missing) `README`